tests/dummy/app/templates/demo/dropdown.hbs

```diff
@@ -1,70 +1,79 @@
 <div class="bs-example">
   <!-- BEGIN-SNIPPET dropdown-demo -->
-  {{#bs-dropdown tagName="span" as |dd|}}
-    {{#dd.toggle}}Dropdown <span class="caret"></span>{{/dd.toggle}}
-    {{#dd.menu as |menu|}}
-      {{#menu.item}}
-        {{#menu.link-to "index"}}Home{{/menu.link-to}}
-      {{/menu.item}}
+  <BsDropdown @tagName="span" as |dd|>
+    <dd.toggle>
+      Dropdown
+      <span class="caret"></span>
+    </dd.toggle>
+    <dd.menu as |menu|>
+      <menu.item>
+        <menu.linkTo>Home</menu.linkTo>
```

Comment:
This is wrong. The positional param has been swallowed. This is basically an extended `{{#link-to}}`, and with its required positional params it cannot be refactored currently into angle bracket syntax. Same for all other instances of `menu.link-to` or `nav.link-to`